### PR TITLE
Add rtic-usb-cdc-echo example

### PR DESCRIPTION
**Pull request by no111u3**

### CODE REVIEW

File: examples/rtic-usb-cdc-echo.rs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            },
+            Local { led },
+            init::Monotonics(mono),
+        )
+    }
+
+    #[idle]
+    fn idle(_: idle::Context) -> ! {
+        loop {}
+    }
```
Comment (dotcypress) [inline examples/rtic-usb-cdc-echo.rs:95]:
```suggestion
```

Comment (dotcypress) [reply examples/rtic-usb-cdc-echo.rs:95]: We can safely remove this part, cause RTIC uses wait loop by default.

**Review by burrbull — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -474,6 +474,10 @@ required-features = ["stm32f411", "rtic"] # stm32f411
 name = "rtic-usart-shell-ssd1306"
 required-features = ["stm32f411", "rtic"] # stm32f411
 
+[[example]]
+name = "rtic-usb-cdc-echo"
+required-features = ["stm32f411", "rtic"] # stm32f411
```
Comment (alexxy) [inline Cargo.toml:479]: add  "otg-fs", "usb_fs" to features

**Pull request merged**